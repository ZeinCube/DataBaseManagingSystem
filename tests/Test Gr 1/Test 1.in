@test "Test 1"
{ 
    @test: "scsc"+"("+")"
    @for(@i:=0; @i<5; @i:=@i+1)
    { 
        @cols := "i1 int unique" 
        @for(@j:=0;@j<@i;@j:=@j+1) 
        { 
           @cols := @cols + ",i"+ @cast @j+2 as string +" int unique"
        } 
        @test: "create table t"+ @cast @i as string +"("+ @cols+")"
    }
    @for( @i:=0;@i<5;@i:=@i+1)
    {
    @test:  "INSERT t2(i1, i2, i3)
             VALUES ("+@i+", "+(@i+2)+", "+-(@i+5)+")"
    }
    @test: "select * from t2"
}