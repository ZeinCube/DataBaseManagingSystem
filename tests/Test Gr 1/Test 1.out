@error: "com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException" "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'scsc()' at line 1"
@executed
@executed
@executed
@executed
@executed
@executed
@executed
@executed
@executed
@executed
@table 
 i1 i2 i3
1 3 -6
2 4 -7
3 5 -8
4 6 -9
@string: "CREATE TABLE `t2` (
  `i1` int(11) DEFAULT NULL,
  `i2` int(11) DEFAULT NULL,
  `i3` int(11) DEFAULT NULL,
  UNIQUE KEY `i1` (`i1`),
  UNIQUE KEY `i2` (`i2`),
  UNIQUE KEY `i3` (`i3`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci"
